src/components/existing-request/existing-request-display.vue

```diff
@@ -407,46 +401,51 @@ export default class ExistingRequestDisplay extends Mixins(
   }
 
   /**
-   * True if the incorporate button should be shown.
+   * True if the Incorporate button should be shown.
    * (It is shown as a distinct button instead of an action.)
    */
-  private get showIncorporateButton (): boolean {
+  get showIncorporateButton (): boolean {
     return (
       this.isBenefitCompany(this.nr) &&
       this.actions.includes(NrAction.INCORPORATE)
     )
   }
 
+  /** True if the Register button should be shown. */
+  get showRegisterButton (): boolean {
+    return this.isFirm(this.nr)
+  }
```

Comment:
This is trivial atm but it might become more complex later, and this also is a nice abstraction from what the component needs to know (do I show the button?) to what the business logic is (is this a firm?).